src/server/game/Entities/Player/Player.cpp

```diff
@@ -7274,6 +7275,14 @@ void Player::RewardReputation(Quest const* quest)
     }
 }
 
+void Player::RewardOtherBonusTalentPoints(uint32 bonusTalentPoints)
+{
+    if (bonusTalentPoints && bonusTalentPoints > 0)
```

Comment:
```cpp
if (bonusTalentPoints)
```

Reply:
@Winfidonarleyan he shouldn't make/use a hook instead of a plain function???

Reply:
> ```c++
> if (bonusTalentPoints)
> ```

@Winfidonarleyan I have fixed it a moment ago. Sorry for my no-computer-can-be-used weekend.